editor.html

```diff
@@ -261,81 +274,14 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
-        'el': 'else do some other thing',
-        'for': 'for each item in a collection of items do something with each item',
-        'try': 'try doing something and handle exceptions (errors)'
-      },
-      'share': {
-        'title': 'Share Code',
-        'instructions': 'Use a password and optional hint (to help you remember the password) to securely create a link to share your code with others.',
-        'passphrase': 'Password: ',
-        'hint': 'Password hint: ',
-        'button': 'Create Link',
-        'description': 'This URL points to your code:',
-        'shortener': 'This is a short version of the link:'
-      },
-      'decrypt': {
-        'title': 'Decrypt Source Code',
-        'instructions': 'Enter a password to decrypt the source code.',
-        'passphrase': 'Password: ',
-        'button': 'Decrypt'
-      },
-      'alerts': {
-        'download': 'Safari has a bug that means your work will be downloaded as an un-named file. Please rename it to something ending in .hex. Alternatively, use a browser such as Firefox or Chrome. They do not suffer from this bug.',
-        'save': 'Safari has a bug that means your work will be downloaded as an un-named file. Please rename it to something ending in .py. Alternatively, use a browser such as Firefox or Chrome. They do not suffer from this bug.',
-        'length': 'Oops! Your script is too long given the limited memory on the device.',
-        'unrecognised_hex': "Sorry, we couldn't recognise this file",
-        'snippets': 'Snippets are disabled when blockly is enabled.'
-      },
-      'confirms': {
-        'quit': "Some of your changes have not been saved. Quit anyway?",
-        'blocks': "You have unsaved code. Using blocks will change your code. You may lose your changes. Do you want to continue?"
-      },
-      'code': {
-        'start': "# Add your Python code here. E.g.\n" +
-                 "from microbit import *\n\n\n" +
-                 "while True:\n" +
-                 "    display.scroll('Hello, World!')\n" +
-                 "    display.show(Image.HEART)\n" +
-                 "    sleep(2000)\n"
-      },
-      'drop': {
-        'python': 'Extracted from a Python file',
-        'hex': 'Extracted from a hex file'
-      },
-      'load': {
-        'title': 'Load Script or HEX file',
-        'instructions': 'Drag and drop a .hex or .py file in here to open it.',
-        'submit': 'Load',
-        'toggle-drop': 'Or drag and drop a file',
-        'toggle-file': 'Or pick a file'
-      }
-    },
-    'flags': {
-      'blocks': false,
-      'snippets': true,
-      'experimental': false,
-      'share': false
-    },
-});
-        </script>
+        <script src="static/js/translation.js"></script>
+  <script>
+      // Call the web_editor function to start the editor running.
+          document.getElementById("lang").onload = function(){
+            web_editor(language);
+            translateEmbedStrings(language);
```

Comment:
If we are passing the language to `web_editor()` anyway, why not execute `translateEmbedStrings(language)` inside `web_editor`?

You could do something like this (from the filesystem branch), and not have to do the "on load" event listener here: https://github.com/bbcmicrobit/PythonEditor/blob/filesystem/python-main.js#L919